src/sc_pipe_management/terra_to_portal_posting.py

```diff
@@ -960,17 +1083,17 @@ def post_all_atac_data_to_portal(terra_data_record: pd.Series, lab: str, award:
                 curr_ctrl_access=curr_ctrl_access,
                 curr_derived_from=all_derived_from,
                 igvf_utils_api=igvf_utils_api,
-                upload_file=upload_file),
+                upload_file=upload_file,
+                output_dir=output_dir),
```

Comment:
We should name these partials so that it is easier to understand what we are doing here.